### ISSUE TITLE
SA1642 doesn't allow static constructors to append text as instance constructors can

### ISSUE THREAD

**Issue by AArnott** [DotNetAnalyzers/StyleCopAnalyzers#3127]
SA1642 allows the standard ctor text to begin with the canned text but then go on to elaborate. But it doesn't allow the same ability on cctor (static constructors).

> Constructor summary documentation should begin with standard text

Note in both ctor and cctor cases, the diagnostic says  "should *begin* with", and yet cctor isn't allowed to merely begin.

**Repro**

In the following class, the ctor is fine, but SA1642 disallows the cctor.

```cs
public class A
{
	/// <summary>
	/// Initializes static members of the <see cref="A"/> class
	/// so that it works.
	/// </summary>
	static A()
	{
	}

	/// <summary>
	/// Initializes a new instance of the <see cref="A"/> class
	/// so that it works.
	/// </summary>
	public A()
	{
	}
}
```

StyleCop version: 1.2.0-beta.164

**Comment by AArnott** [comment]
It appears that if I add a `.` (period) after "class" in the cctor that the warning goes away. So perhaps the cctor required starter text inappropriately includes the trailing `.` where the ctor required starter text appropriately omits that trailing `.`
